s4e-backend/src/test/resources/geoserver/datastore.properties

```diff
@@ -0,0 +1,5 @@
+SPI=org.geotools.data.postgis.PostgisNGJNDIDataStoreFactory
```

Comment:
Do you need these resources here? Wouldn't it be enough to use the ones from the main resources? Aren't they the same?